### PR TITLE
WSTEAM1-280: Add media-page identifier to ATI tracking.

### PR DESCRIPTION
**Pull request by shayneahchoon**
Resolves JIRA [WSTEAM1-280](https://jira.dev.bbc.co.uk/browse/WSTEAM1-280)

Screenshot 2023-03-07 at 13 03 05

Screenshot 2023-03-07 at 13 03 17

Screenshot 2023-03-07 at 13 03 21

Overall changes
======
Adds media-page identifier to ATI so that MediaPage sends tracking information adjacent to that of Article Pages. 

Code changes
======

- _ATI Tracking functions for building the Article URL has been modified to accommodate the media-article page type._
- _EventTracking has been modified to accommodate tracking requests for the media-article page type._

Testing
======
1. Open a MediaArticlePage.
2. Open the network tab in chrome. 
3. ATI tracking requests should be made similar to that of ArticlePages, but with 'media-article' set as the x7 header. 

Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

### CODE REVIEW

File: src/app/contexts/EventTrackingContext/index.jsx [hunk 1]
```diff
@@ -27,6 +28,7 @@ export const EventTrackingContext = createContext({});
 const getCampaignID = pageType => {
   const campaignID = {
     [ARTICLE_PAGE]: 'article',
+    [MEDIA_ARTICLE_PAGE]: 'media-article',
```
Comment (amoore108) [inline src/app/contexts/EventTrackingContext/index.jsx]: I believe this should be called `article-sfv` in analytics, just to confuse things 😅

Comment (shayneahchoon) [reply src/app/contexts/EventTrackingContext/index.jsx]: Soz, I forgot to change it back.

**Review by amoore108 — APPROVED** [review]
Nice work 👏 LGTM.

**Review by Isabella-Mitchell — APPROVED** [review]
Looks good!

**Review by mfonofm — APPROVED** [review]
Niice!

**Pull request merged**